app/Http/Controllers/Auth/PasswordResetController.php

```
@@ -0,0 +1,104 @@
+            Mail::raw(
+                "Reset your osTicket password:\n\n{$resetUrl}\n\nThis link expires in 60 minutes.",
+                fn ($message) => $message
+                    ->to($staff->email)
+                    ->subject('Reset Your Password')
+            );
+        }
+
+        return back()->with('status', 'If an account exists with that email, a reset link has been sent.');
+    }
+
+    public function showResetForm(Request $request, string $token): Response|RedirectResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)